pkg/daemon/update.go

```diff
@@ -1126,10 +1131,11 @@ func (dn *Daemon) updateFiles(oldIgnConfig, newIgnConfig ign3types.Config) error
 
 func restorePath(path string) error {
 	if out, err := exec.Command("cp", "-a", "--reflink=auto", origFileName(path), path).CombinedOutput(); err != nil {
-		return errors.Wrapf(err, "restoring %q from orig file %q: %s", path, origFileName(path), string(out))
+		return fmt.Errorf("restoring %q from orig file %q: %s: %w", path, origFileName(path), string(out), err)
 	}
 	if err := os.Remove(origFileName(path)); err != nil {
-		return errors.Wrapf(err, "deleting orig file %q: %v", origFileName(path), err)
+		errs := kubeErrs.NewAggregate([]error{err, err})
```

Comment:
Why are you aggregating the same error twice? You should be able to use `fmt.Errorf` since there's only one error here.